index.d.ts

```diff
@@ -0,0 +1,7 @@
+namespace sortOn {
+	type Property<T> = string | string[] | ((element: T) => any) | ((element: T) => any)[];
```

Comment:
`any` => `unknown`

Reply:
Yeah, I was thinking "accept `any` and return `unknown`", but didn't think this thru all the way 😄 

Since this is a function that's passed in, that function should return `unknown` as to not make any assumptions about the return value 👌